browser/farbling/brave_screen_farbling_browsertest.cc

```diff
@@ -249,24 +297,47 @@ class BraveScreenFarblingBrowserTest : public InProcessBrowserTest {
         if (!allow_fingerprinting && !IsFlagDisabled()) {
           EXPECT_GE(child_bounds.x(), parent_bounds.x());
           EXPECT_GE(child_bounds.y(), parent_bounds.y());
-          EXPECT_GE(10 + parent_bounds.width(), child_bounds.width());
-          EXPECT_GE(10 + parent_bounds.height(), child_bounds.height());
+          int maxWidth = 10 + std::max(450, parent_bounds.width());
+          int maxHeight = 10 + std::max(450, parent_bounds.width());
+          EXPECT_GE(maxWidth, child_bounds.width());
+          EXPECT_GE(maxHeight, child_bounds.height());
         } else {
           EXPECT_LE(child_bounds.x(), std::max(80, 10 + parent_bounds.x()));
           EXPECT_LE(child_bounds.y(), std::max(80, 10 + parent_bounds.y()));
           EXPECT_LE(parent_bounds.width(), child_bounds.width());
           EXPECT_LE(parent_bounds.height(), child_bounds.height());
         }
+        if (!test_iframe) {
+          auto bounds_before = popup->window()->GetBounds();
+          auto* widget = views::Widget::GetWidgetForNativeWindow(
+              popup->window()->GetNativeWindow());
+          auto waiter = WidgetBoundsChangeWaiter(widget, 10);
+          ASSERT_TRUE(
+              ExecJs(popup_contents, "moveTo(screenX + 11, screenY + 12)"));
+          ASSERT_TRUE(ExecJs(popup_contents,
+                             "resizeTo(outerWidth + 13, outerHeight + 14)"));
+          waiter.Wait();
+          auto bounds_after = popup->window()->GetBounds();
+          EXPECT_EQ(11, bounds_after.x() - bounds_before.x());
+          EXPECT_EQ(12, bounds_after.y() - bounds_before.y());
+          EXPECT_EQ(13, bounds_after.width() - bounds_before.width());
+          EXPECT_EQ(14, bounds_after.height() - bounds_before.height());
+        }
       }
     }
   }
 
+  GURL& IframeUrl() { return iframe_url_; }
```

Comment:
nit: these simple getters would normally be `iframe_url()`, `top_url()`, etc...

Reply:
fixed